package.json

```diff
@@ -23,12 +23,12 @@
   },
   "devDependencies": {
     "chai": "^2.3.0",
-    "gulp": "^3.8.6",
+    "gulp": "^3.9.1",
```

Comment:
Please remove this file from this PR, it is unrelated to the addition of support for Solidity.